website/content/api-docs/secret/ssh.mdx

```diff
@@ -170,7 +170,9 @@ This endpoint creates or updates a named role.
 
   ~> **Note**: In FIPS 140-2 mode, the following algorithms are not certified
      and thus should not be used: `ed25519`.
-
+- `allow_empty_principals` `(bool: false)` - Allow signing certificates with
```

Comment:
I am not sure if the absence of the empty line matters or not. The Vercel step was skipped by the PR build, so I couldn't verify it.